### ISSUE TITLE
$authenticated doesn't work with v2.7.4  and ldap plugin

### ISSUE THREAD

**Issue by kopax** [verdaccio/verdaccio#696]
I am trying to set `access: $authenticated` to the scope `'@someorg/*':` but I keep getting authenticated then the next request is anonymous.

Does anybody know about such a bug ? 

I am using verdaccion v2.7.4 , this is my configuration:

```yml
storage: /verdaccio/storage

auth:
  ldap:
    type: ldap
    groupNameAttribute: 'cn'
    client_options:
      url: "ldap://ldap.example.com"
      adminDn: "cn=readonly,dc=example,dc=com"
      adminPassword: "example"
      searchBase: "dc=example,dc=com"
      searchFilter: "(&(uid={{username}})(memberOf=cn=npm_users,ou=groups,dc=example,dc=com))"
      cache: False
      searchAttributes:
        - "*"
        - memberOf
      tlsOptions:
        rejectUnauthorized: False
    max_users: -1
  htpasswd:
    file: ./htpasswd
#   # Maximum amount of users allowed to register, defaults to "+inf".
#   # You can set this to -1 to disable registration.
#   # currently used only for deployment accounts
    max_users: 2

# a list of other known repositories we can talk to

uplinks:
  npmjs:
    url: https://registry.npmjs.org/

packages:
  '@someorg/*':
    # scoped packages
    access: $authenticated
    publish: npm_users
#    proxy: npmjs

  '@*/*':
    # scoped packages
    access: $all
    publish: $authenticated

    # if package is not available locally, proxy requests to 'npmjs' registry
    proxy: npmjs

  '**':
    # allow all users (including non-authenticated users) to read and
    # publish all packages
    #
    # you can specify usernames/groupnames (depending on your auth plugin)
    # and three keywords: "$all", "$anonymous", "$authenticated"
    access: $all

    # allow all known users to publish packages
    # (anyone can register by default, remember?)
    publish: $authenticated

    # if package is not available locally, proxy requests to 'npmjs' registry
    proxy: npmjs

# log settings
logs:
  - {type: stdout, format: pretty, level: http}
  #- {type: file, path: verdaccio.log, level: info}
web:
  enable: true
  title: Redacted
```

I am able to publish a scoped package in `@someorg/*` but no user can download it after:

```bash
 http <-- 200, user: dga(115.73.7.165 via 172.16.0.1), req: 'GET /@someorg%2fdeleteme', bytes: 0/3248
 http <-- 403, user: undefined(115.73.7.165 via 172.16.0.1), req: 'GET /@someorg%2fdeleteme/-/deleteme-0.0.4.tgz', error: unregistered users are not allowed to access package @someorg/deleteme

```

**Comment by kopax** [comment]
@juanpicado I am sorry to ask that but can we leave this open? I know many verdaccio users are actually having their ldap working with group. All I need is one discussion with someone who succeeded such configuration.

Right now we have not been able to configure the security layer properly and we are struggling between version to have one. If we can't set it up we will have to wait until we have a solution that works with ldap. 

Verdaccio seems fine to me but I feel bad trying to make it work with luck.

**Comment by kopax** [comment]
In fact, @juanpicado I finally got information from the author of `verdaccio-ldap`. It was solved by using the fix release in 2.0.0 (fixing 1.4.0 bringing ldap group revamp).

Thanks for your concern, I'll close again. 

Verdaccio is a must have, so as the LDAP plugin.

**Comment by juanpicado** [comment]
Cool yw 🤓  Perhaps you ... https://github.com/verdaccio/docker-examples/issues/7 :octocat:  ...
